app/views/users/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<% provide :title, t('.signup') %>
+<h1 class="center"><%= t ".register" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
```

Comment:
form for là gì? có loại form nào khác nữa không?

Reply:
dạ form_for  có nghĩa là phương thức tạo ra một form cho để có thể create hoặc update các thuộc tính của một model hay object cụ thể.
Dạ ngoài ra còn các loại như form_with và form_tag